plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduRecordCursor.java

```diff
@@ -101,31 +98,16 @@ private int mapping(int field)
     @Override
     public boolean advanceNextPosition()
     {
-        boolean needNextRows = !started || !nextRows.hasNext();
-
         if (!started) {
+            kuduScannerIterator = scanner.iterator();
```

Comment:
Can we assign it in the constructor of `KuduRecordCursor` then we might not need the `started` flag